pkg/jobrunaggregator/jobtableprimer/job_typer.go

```diff
@@ -80,6 +81,10 @@ func newJob(name string) *jobRowBuilder {
 	if len(versions) >= 1 {
 		currRelease = versions[0]
 	}
```

Comment:
could be an else too, if we don't detect a version then panic.

Reply:
pushed